asynchronous/callbacks.js

```diff
@@ -0,0 +1,19 @@
+// Create an Array
+const myNumbers = [4, 1, -20, -7, 5, 9, -6];
+
+// Call removeNeg with a callback
+const posNumbers = removeNeg(myNumbers, (x) => x >= 0);
+
+// Display Result
+console.log(posNumbers);
+
+// Keep only positive numbers
+function removeNeg(numbers, callback) {
+  const myArray = [];
+  for (const x of numbers) {
+    if (callback(x)) {
```

Comment:
If the callback doesn't return anything then the "if" condition will not be met and hence the numbers will not be pushed into the array. To prevent this, a return type check can be done before the loop and error message can be logged.